includes/custom-measurements-disclaimer.mdx

```diff
@@ -0,0 +1,5 @@
+<Alert level="warning">
+
+For a more flexible solution where your measurements are not tied to transactions and not limited to 10 per transaction, we recommend you use [custom metrics](https://docs.sentry.io/product/metrics/) instead.
```

Comment:
```suggestion
Custom performance measurements are limited to 10 per transaction. If you have more than 10 measurements per transaction, the list will be truncated. For a more flexible solution, we recommend using [custom metrics](https://docs.sentry.io/product/metrics/) instead.
```

Reply:
We should rewrite this since metrics will eventually be tied to spans

Reply:
applied 👍